### PR TITLE
Update usage section in README

### PR DESCRIPTION
**Pull request by matthiasschaub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,78 +9,39 @@
...
 [![status: active](https://github.com/GIScience/badges/raw/master/status/active.svg)](https://github.com/GIScience/badges#active)
 
-
-## Vision
-
-What is OQT?
-* A project that **formalizes the knowledge on OSM data quality** within HeiGIT and the GIScience Research Group.
-* A tool that end users, e.g. humanitarian organisations and public administration, can use to get information on the **quality of OSM data for their specific region and use-case**.
-* A **web app** that builds upon the existing infrastructure, especially [ohsomeHeX](https://ohsome.org/apps/osm-history-explorer) and the [ohsome dashboard](https://ohsome.org/apps/dashboard).
-* A **data integration tool**, that brings together the implementation of a variety of intrinsic and extrinsic data quality metrics.
```
Comment (matthiasschaub) [inline README.md:19]: Should this be in the README?

File: README.md [hunk 2]
```diff
@@ -9,78 +9,39 @@
...
+The easiest access to quality reports is through the OQT [website](https://oqt.ohsome.org). There reports with pre-computed indicators can be accessed for a selection of regions made by the OQT team. But it is not possible to upload custom areas-of-interest (AOIs) and compute report or indicators for those.
 
+Requests for individual indicators as well as reports can be made to the OQT [API](https://oqt.ohsome.org/api/docs). Retrievable are pre-computed indicators for selected regions (Parameters `dataset=regions` and `featureId`) as well as for custom AOIs (Parameter `bpolys`). The major drawbag is the limitation of the input geometry size. For more information check out the interactive API [documentation](https://oqt.ohsome.org/api/docs).
 
-## OQT Components
+The last way to use OQT is by setting up OQT on your local machine and use the provided CLI. This is the most flexible way but some technical knowledge is required to set it up. For now please refer to the development setup [documentation](/docs/development_setup.md) on how to setup OQT localy on your machine. After installation of the `ohsome_quality_analyst` Python package run `oqt --help` for documation about the CLI.
```
Comment (joker234) [inline README.md:41]: The development setup does not cover how you can setup the CLI as the README did.

Comment (joker234) [inline README.md:41]:
```suggestion
The last way to use OQT is by setting up OQT on your local machine and use the provided CLI. This is the most flexible way but some technical knowledge is required to set it up. For now please refer to the [development setup documentation](/docs/development_setup.md) on how to set up OQT locally on your machine. After installation of the `ohsome_quality_analyst` Python package run `oqt --help` for documentation about the CLI.
```

File: README.md [hunk 3]
```diff
@@ -9,78 +9,39 @@
...
-  create-report          Create a Report and print results to stdout.
-  list-datasets          List available datasets.
-  list-indicators        List available indicators and their metadata.
-  list-layers            List available layers and how they are definied...
-  list-regions           List available regions.
-  list-reports           List available reports and their metadata.
-```
+The easiest access to quality reports is through the OQT [website](https://oqt.ohsome.org). There reports with pre-computed indicators can be accessed for a selection of regions made by the OQT team. But it is not possible to upload custom areas-of-interest (AOIs) and compute report or indicators for those.
 
+Requests for individual indicators as well as reports can be made to the OQT [API](https://oqt.ohsome.org/api/docs). Retrievable are pre-computed indicators for selected regions (Parameters `dataset=regions` and `featureId`) as well as for custom AOIs (Parameter `bpolys`). The major drawbag is the limitation of the input geometry size. For more information check out the interactive API [documentation](https://oqt.ohsome.org/api/docs).
```
Comment (joker234) [inline README.md:39]:
```suggestion
Requests for individual indicators as well as reports can be made to the OQT [API](https://oqt.ohsome.org/api/docs). Retrievable are pre-computed indicators for selected regions (Parameters `dataset=regions` and `featureId`) as well as for custom AOIs (Parameter `bpolys`). The major drawback is the limitation of the input geometry size. For more information check out the interactive [API documentation](https://oqt.ohsome.org/api/docs).
```

**Review by joker234 — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -9,78 +9,39 @@
...
-
-The response schema is defined here: [docs/api.md](/docs/api.md)
-
+## OQT Components
 
-### CLI
+This diagram gives an overview of the OQT components:
 
-To use the OQT CLI the `ohsome_quality_analyst` Python package needs to be installed and configured. Additionally access to the OQT database is required. Please reach out if you need one.
+![](/docs/img/UML-Component-Diagram.png)
```
[the remark this answers is not in the document]
Comment (matthiasschaub) [reply README.md:27]: Put it to the bottom.

File: README.md (hunk 2 above)
Comment (matthiasschaub) [reply README.md]: Yes and no. The problem here is that for the CLI to work one must set up a the database and configure OQT accordingly. Currently all those information's are found in the dev docs. 

The CLI set up described here in the README was how to install OQT CLI in theory but a lot of details for getting it to run were missing.

File: README.md (hunk 1 above)
Comment (matthiasschaub) [reply README.md:19]: This is also on Confluence.

**Review by joker234 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Hagellach37) [reply README.md:19]: Hey,
I think we do not need to state it as the vision, but for sure need 2-3 general sentences what OQT is about. But what you have below might be already enough.

so yes, let's remove it from the readme.

**Pull request merged**